akka-http-core/src/main/scala/akka/http/Http.scala

```diff
@@ -60,13 +60,13 @@ class HttpExt(config: Config)(implicit system: ActorSystem) extends akka.actor.E
    * connections are being accepted at maximum rate, which, depending on the applications, might
    * present a DoS risk!
    */
-  def bindAndstartHandlingWith(handler: Flow[HttpRequest, HttpResponse, _],
+  def bindAndStartHandlingWith(handler: Flow[HttpRequest, HttpResponse, _],
```

Comment:
there was also the question in #16972 on if this method should be called `bindAndHandle` to be more similar to TCP.
// cc @rkuhn @jrudolph @hseeberger @sirthias